sdk/webpubsub/azure-messaging-webpubsubservice/README.md

```diff
@@ -43,14 +43,48 @@ python -m pip install azure-messaging-webpubsubservice
 
 ### Authenticating the client
 
-In order to interact with the Azure WebPubSub service, you'll need to create an instance of the [WebPubSubServiceClient][webpubsubservice_client_class] class. In order to authenticate against the service, you need to pass in an AzureKeyCredential instance with endpoint and api key. The endpoint and api key can be found on the azure portal.
+#### 1. Create with an API Key Credential
+
+You can get the [API key][api_key] or [Connection string][connection_string] in the [Azure Portal][azure_portal].
+Once you have the value for the API key, you can pass it as a string into an instance of [AzureKeyCredential][azure-key-credential]. 
+Use the key as the credential parameter to authenticate the client:
 
 ```python
 >>> from azure.messaging.webpubsubservice import WebPubSubServiceClient
 >>> from azure.core.credentials import AzureKeyCredential
->>> client = WebPubSubServiceClient(endpoint='<endpoint>', credential=AzureKeyCredential('somesecret'))
->>> client
-<WebPubSubServiceClient endpoint:'<endpoint>'>
+
+>>> client = WebPubSubServiceClient(endpoint='<endpoint>', credential=AzureKeyCredential("<api_key>"))
+```
+
+Once you have the value for the connection string, you can pass it as a string into the function `from_connection_string` and it will 
```

Comment:
I think we can do a separate section for `from_connection_string`, because there are 3 ways to authenticate to the clien
1. `client = WebPubSubServiceClient(AzureKeyCredential())`
2. `client = WebPubSubServiceClient.from_connection_string()`
3. `client = WebpubSubServiceClient(DefaultAzureCredential())`
So it seems fair to have 3 subsections here

Reply:
Actually, 1 and 2 is the same authentication way since we will extract `endpoint` and `access key` from `connection_string`. Customers could also get them in the same portal page and that's why I put them in the same section.